OSDOCS#6792: Azure: Document installing OpenShift in a disconnected environment (UPI) (#63336)
openshift/openshift-docs: installing/installing_azure/installing-restricted-networks-azure-user-provisioned.adoc

:_content-type: ASSEMBLY
[id="installing-restricted-networks-azure-user-provisioned"]
= Installing a cluster on Azure in a restricted network with user-provisioned infrastructure
include::_attributes/common-attributes.adoc[]
:context: installing-restricted-networks-azure-user-provisioned

toc::[]

In {product-title} version {product-version}, you can install a cluster on Microsoft Azure by using infrastructure that you provide.

Several link:https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/overview[Azure Resource Manager] (ARM) templates are provided to assist in completing these steps or to help model your own.

[IMPORTANT]
====

> The steps for performing a user-provisioned infrastructure installation are provided as an example only. Installing a cluster with infrastructure you provide requires knowledge of the cloud provider and the installation process of {product-title}. Several ARM templates are provided to assist in completing these steps or to help model your own. You are also free to create the required resources through other methods; the templates are just an example.

Reviewer: The last part of the admonition isn't needed, suggestion to remove it:
```suggestion
The steps for performing a user-provisioned infrastructure installation are provided as an example only. Installing a cluster with infrastructure you provide requires knowledge of the cloud provider and the installation process of {product-title}. Several ARM templates are provided to assist in completing these steps or to help model your own. You are also free to create the required resources through other methods.
```

====

.Prerequisites

* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
* You xref:../../installing/installing_azure/installing-azure-account.adoc#installing-azure-account[configured an Azure account] to host the cluster and determined the tested and validated region to deploy the cluster to.

> * You xref:../../installing/disconnected_install/installing-mirroring-installation-images.adoc#installation-about-mirror-registry_installing-mirroring-installation-images[mirrored the images for a disconnected installation] to your registry and obtained the imageContentSources data for your version of {product-title}.

Reviewer: ```suggestion
* You xref:../../installing/disconnected_install/installing-mirroring-installation-images.adoc#installation-about-mirror-registry_installing-mirroring-installation-images[mirrored the images for a disconnected installation] to your registry and obtained the `imageContentSources` data for your version of {product-title}.
```

+
[IMPORTANT]
====

> Because the installation media is on the mirror host, you can use that computer to complete all installation steps.

Reviewer: are they required to use it? the IMPORTANT admonition makes me think they are, in which case I think that should be made clear:
```suggestion
Because the installation media is on the mirror host, you must use that computer to complete all installation steps.
```
But if not, consider using a NOTE admonition instead

====
* If you use a firewall, you xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured it to allow the sites] that your cluster requires access to.

> * If the cloud identity and access management (IAM) APIs are not accessible in your environment, or if you do not want to store an administrator-level credential secret in the `kube-system` namespace, you can xref:../../installing/installing_azure/manually-creating-iam-azure.adoc#manually-creating-iam-azure[manually create and maintain IAM credentials].

Reviewer: the other prereqs are in past tense:
```suggestion
* If the cloud identity and access management (IAM) APIs are not accessible in your environment, or if you do not want to store an administrator-level credential secret in the `kube-system` namespace, you have xref:../../installing/installing_azure/manually-creating-iam-azure.adoc#manually-creating-iam-azure[manually created and maintain IAM credentials].
```

:_content-type: ASSEMBLY
[id="installing-restricted-networks-azure-user-provisioned"]
= Installing a cluster on Azure in a restricted network with user-provisioned infrastructure
include::_attributes/common-attributes.adoc[]
:context: installing-restricted-networks-azure-user-provisioned

toc::[]

In {product-title} {product-version}, you can install a cluster on Microsoft Azure that uses infrastructure that you provide and an internal mirror of the installation release content.

[IMPORTANT]
====
You can install an {product-title} cluster by using mirrored installation release content, but your cluster requires internet access to use the Azure APIs.
====

Several link:https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/overview[Azure Resource Manager] (ARM) templates are provided to assist in completing these steps or to help model your own. You can create the required resources through other methods.

[IMPORTANT]
====
The steps for performing a user-provisioned infrastructure installation are provided as an example. Installing a cluster with infrastructure you provide requires knowledge of the cloud provider and the installation process of {product-title}.

> ====

Reviewer 2: I had used the existing docs for user-provisioned installation. You can follow the same from here: https://docs.openshift.com/container-platform/4.13/installing/installing_azure/installing-azure-user-infra.html

.Prerequisites

* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].

> * You xref:../../installing/installing_azure/installing-azure-account.adoc#installing-azure-account[configured an Azure account] to host the cluster and determined the tested and validated government region to deploy the cluster to.

Reviewer 2: ```suggestion
* You xref:../../installing/installing_azure/installing-azure-account.adoc#installing-azure-account[configured an Azure account] to host the cluster and determined the tested and validated region to deploy the cluster to.
```
The work`government` is not needed.
